package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "dev": "node server.js",
     "build": "next build",
-    "start": "NODE_ENV=production node server.js",
```

Comment:
does Heroku set it by default? it affects certain parts of the code we haven't had any problems running it locally before - I think "dev" key is made for that?

Reply:
When trying to start up the node wrote that "NODE_ENV" isn't an internal or external command, an executable program or a batch file.

Reply:
> does Heroku set it by default? it affects certain parts of the code we haven't had any problems running it locally before - I think "dev" key is made for that?

When trying to start up the node wrote that "NODE_ENV" isn't an internal or external command, an executable program or a batch file.

Reply:
My point is that this env variable is set there for a reason in the production environment. We need to make sure that we are not breaking the production environment behavior with this change (Heroku is where it's being deployed to). 

Are you running this on Windows? Would `npm run-script dev` be enough?